include/cantera/oneD/IonFlow.h

```diff
@@ -67,12 +67,37 @@ class IonFlow : public StFlow
                               vector<double>& mobi_e);
 
 protected:
-    /*!
-     * This function overloads the original function. The residual function
-     * of electric field is added.
+
+    /**
+     * @brief Evaluate the electric field equation residual by Gauss's law.
+     *
+     * The function calculates the electric field equation as:
+     * \f[
+     *    \frac{dE}{dz} = \frac{e}{\varepsilon_0} \sum (q_k \cdot n_k)
+     * \f]
```

Comment:
In order to differentiate from $\LaTeX$, Cantera uses doxygen comments starting with `@` (here and elsewhere)
```suggestion
     * @f[
     *    \frac{dE}{dz} = \frac{e}{\varepsilon_0} \sum (q_k \cdot n_k)
     * @f]
```